packages/astro/templates/env.mjs

```diff
@@ -2,14 +2,19 @@
 import { schema } from 'virtual:astro:env/internal';
 import {
 	createInvalidVariablesError,
-	getEnv,
+	getEnv as _getEnv,
 	getEnvFieldType,
 	setOnSetGetEnv,
 	validateEnvVariable,
 } from 'astro/env/runtime';
 
+const getEnv = (key) => {
+	// @@GET_ENV@@
+	return _getEnv(key);
```

Comment:
This code will end up with two `return` statements, which isn't the best. Maybe we can leave the placeholder and add an `else` clause to the `if` we have `vite-plugin-env.ts`, line 157, which will replace the placeholder with `return _getEnv(key)`